tmt/export.py

```diff
@@ -61,6 +61,42 @@ def _nitrate_find_fmf_testcases(test):
             pass
 
 
+def markdown_to_html(text_field, test_md):
+    """ Convert markdown to html """
+    try:
+        import markdown
+    except ImportError:
+        raise ConvertError("Install tmt-test-convert to export tests.")
+
+    try:
+        with open(test_md, 'r') as f:
+            text = f.read()
+            html = markdown.markdown(text)
+    except IOError:
+        raise ConvertError(f"Unable to open '{test_md}'.")
+
+    html_splitlines = html.splitlines()
+    result = str()
+    i = 0
+    while html_splitlines:
+        try:
+            if re.search("^<h[1-4]>" + text_field + "</h[1-4]>$", html_splitlines[i]):
+                for j in range(i+1, len(html_splitlines)):
+                    if re.search("^<h[1-4]>(.+?)</h[1-4]>$", html_splitlines[j]):
+                        # Line to separate text fields under the same tag
+                        # if there are 2 or more text fields
+                        if html_splitlines.count(html_splitlines[i]) >= 2:
+                            result += '-' * 80
```

Comment:
So we convert `markdown` to `html`. What about using the `<hr/>` tag instead of the ascii dash separator?